### ISSUE TITLE
Handle rdf:plainliteral in the exported SHACL

### ISSUE THREAD

**Issue by cristianvasquez** [OP-TED/model2owl#219]
The ePO ontology produces SHACL with references to rdf:plainliteral, that is either a `xsd:string` or `rdf:langString` like in this example

```turtle
core-shape:org-Organization-epo-hasLegalFormType a sh:PropertyShape ;
    rdfs:isDefinedBy core-shape:core-shape ;
    sh:datatype rdf:PlainLiteral ;
    sh:maxCount 1 ;
    sh:name "Has legal form type" ;
    sh:path :hasLegalFormType .
```

However, this is not recognized by most SHACL engines

A mechanism is needed to export SHACL that can be used by such engines.

One option is to export something like this:

```rdf
core-shape:org-Organization-epo-hasLegalFormType a sh:PropertyShape ;
    rdfs:isDefinedBy core-shape:core-shape ;
    sh:maxCount 1 ;
    sh:name "Has legal form type" ;
    sh:path :hasLegalFormType ;
    sh:or (
        [ sh:datatype xsd:string ]
        [ sh:datatype rdf:langString ]
    ) .
```

Alternatively declare a `sh:NodeShape`, in this case, the a `a4g_shape:PlainLiteral` 

```
a4g_shape:PlainLiteral a sh:NodeShape ;
    sh:or (
        [ sh:datatype xsd:string ]
        [ sh:datatype rdf:langString ]
    ) .
```

So it can be used as

```
sh:node a4g_shape:PlainLiteral

```

Related: https://github.com/OP-TED/ted-rdf-mapping-eforms/issues/20

**Comment by cristianvasquez** [comment]
related: https://github.com/OP-TED/ted-rdf-mapping/issues/407

**Comment by gkostkowski** [comment]
The solution was implemented and tested (PR #233). The change will be included in the next model2owl release.

**Solution details:**

The first option recommended by @cristianvasquez was implemented.

An example demonstrating the change:

Input excerpt (XMI):

```xml
<element xmi:idref="EAID_29678A10_6473_461a_BE84_76269B5BBD68" xmi:type="uml:Class" name="locn:Address" scope="public">
...
    <attributes>
        <attribute xmi:idref="EAID_829CA5E1_0FDA_4c48_8F2E_DB19DABE380D" name="locn:fullAddress" scope="Public">
            ...
	    <properties type="rdf:PlainLiteral"/>
        </attribute>
    </attributes>
</element>
```

Output excerpt (SHACL shape):

```xml
core-shape:locn-Address-locn-fullAddress a sh:PropertyShape ;
    ...
    sh:or ( [ sh:datatype xsd:string ] [ sh:datatype rdf:langString ] ) .
```


The change affects the SHACL artefact in both RDF/XML and TTL serialization formats.

The new behaviour is active by default. The extension is configurable and can be disabled if needed by setting the new [translatePlainLiteralToStringTypesInSHACL](https://github.com/OP-TED/model2owl/pull/233/files#diff-b8d5f798c186c9ee4faeca59e6b819e403b1315e3899733ddcfa24ea30508391R106) configuration parameter to `fn:false()`.
